grapher/stackedCharts/StackedDiscreteBarChart.tsx

```diff
@@ -325,63 +326,77 @@ export class StackedDiscreteBarChart
                     bounds={innerBounds}
                 />
                 <HorizontalCategoricalColorLegend manager={this} />
-                {this.items.map(({ label, bars }) => {
-                    // Using transforms for positioning to enable better (subpixel) transitions
-                    // Width transitions don't work well on iOS Safari – they get interrupted and
-                    // it appears very slow. Also be careful with negative bar charts.
-                    const tooltipProps = {
-                        label,
-                        bars,
-                        targetTime: this.manager.endTime,
-                        timeColumn: this.inputTable.timeColumn,
-                        formatColumn: this.formatColumn,
-                    }
+                <Flipper flipKey={this.items.map((i) => i.label)} element="g">
+                    {this.items.map(({ label, bars }) => {
+                        // Using transforms for positioning to enable better (subpixel) transitions
+                        // Width transitions don't work well on iOS Safari – they get interrupted and
+                        // it appears very slow. Also be careful with negative bar charts.
+                        const tooltipProps = {
+                            label,
+                            bars,
+                            targetTime: this.manager.endTime,
+                            timeColumn: this.inputTable.timeColumn,
+                            formatColumn: this.formatColumn,
+                        }
 
-                    const result = (
-                        <g
-                            key={label}
-                            className="bar"
-                            transform={`translate(0, ${yOffset})`}
-                        >
-                            <TippyIfInteractive
-                                lazy
-                                isInteractive={
-                                    !this.manager.isExportingtoSvgOrPng
-                                }
-                                hideOnClick={false}
-                                content={
-                                    <StackedDiscreteBarChart.Tooltip
-                                        {...tooltipProps}
-                                    />
-                                }
+                        // This custom spring is very gentle and doesn't wobble.
+                        const spring = { stiffness: 130, damping: 23 }
```

Comment:
It feels to me like it's _too_ gentle, but curious [what others on Slack think](https://owid.slack.com/archives/CQQUA2C2U/p1623085541028300).